nndt/space2/plot_tree.py

```diff
@@ -96,3 +105,4 @@ def _plot(
         pl.show(cpos=cpos)
     else:
         pl.show(screenshot=filepath, cpos=cpos)
+
```

Comment:
seems you don't use linter, please configure the pre-commit hooks

Reply:
fix